### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by Allenkeys**
### Project requirements

- Use JavaScript `classes` instead of `objects` and `arrays`.
  - You can use objects and arrays, but do it *inside* classes.
- Create class `methods` to add and remove books.
- Maintain all of the original functionality of the application (i.e. no bugs were introduced while refactoring).
- Don't use any library or framework, use only plain JavaScript.
- Add CSS styles to the application to make it match this wireframe:
- You can use the styles of your choice (colors, fonts, backgrounds, etc.), but try to follow the general layout.
- Concentrate on the general layout, rather than the details.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,65 +1,78 @@
...
+class Render {
+  collections = [];
+
+  static addBookToCollection(title, author) {
+    const book = new Book(title, author);
+    this.collections = this.collections.concat(book);
+    console.log(book);
+    addToLocalStorage(this.collections);
+    return this.collections;
+  }
```
Comment (DeliceLydia) [inline main.js]: - [ ] I am not able to add a book please check the error I am getting and fix that
Screenshot from 2022-10-26 21-28-47

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! highlights:
- Good job on using javascript classes
- Good README


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: main.js (hunk 1 above)
Comment (Allenkeys) [reply main.js]: Thanks for the feedback :bow: 
While it was working on my machine cos I already had data stored in the local storage, this led to a silent error on any other machine.

With you feedback, I've been able to get it fixed. :+1:

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 
- Good overall design ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**